ibis/common/patterns.py

```diff
@@ -562,6 +609,10 @@ def __new__(cls, target):
         else:
             return Apply(target)
 
+    # TODO(kszucs): to support As[int] or CoercedTo[int] syntax
```

Comment:
Can we delete or address this, or maybe keep the TODO and remove the commented code?

Reply:
Removing the commented code.